app/javascript/src/application.scss

```diff
@@ -15,10 +17,24 @@ $govuk-assets-path: '../../../node_modules/govuk-frontend/assets/';
   background-color: govuk-colour("grey-4");
 }
 
+.govuk-\!-display-inline-block {
+  display: inline-block !important;
+}
+
+.flex {
+  flex: 1;
+}
```

Comment:
Is this used? I can't see it elsewhere in this PR. Flex is supported by IE11, but has a bunch of bugs we need to be careful of (https://caniuse.com/#search=flex)

Reply:
Leftover, removed